cmd/interactive.go

```diff
@@ -485,6 +485,9 @@ func buildModelfile(opts runOptions) string {
 	}
 
 	for _, msg := range opts.Messages {
+		if strings.Contains(msg.Content, "\"") {
```

Comment:
A test for the triple quotes here would be great https://github.com/ollama/ollama/blob/main/cmd/interactive_test.go#L57

Reply:
I thought about putting that in, but all this is rewritten in the new changes to the Create API which hopefully we'll get in in the next few days.